bin/commands/sample/sub/.help

```diff
@@ -1 +1,7 @@
-A sample sub-command.
+Synopsis
+    zwe sample sub [sub-command] [parameter [parameter]...]
```

Comment:
Question for @jackjia-ibm actually: shouldn't we hide this from an end-user? Extensions aren't going to add commands to `zwe` right, so this command only helps us core zowe devs. May be better as documentation in a readme or something.

Reply:
With original idea, I do think we may allow extenders to expand `zwe` command. I remember JoeW also thought about this. The thought was listed in the initial proposal https://github.com/zowe/zowe-install-packaging/issues/2435#issuecomment-983931109 page 3. But it's not priority and we have so many topics, so we didn't really discuss details.

Reply:
Ok, we should talk about this more sometime later. I'm unsure how much we should do here without just telling people to use the Zowe CLI.